src/libraries/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/Context.cs

```diff
@@ -104,8 +104,7 @@ private bool BindSam(string target, string userName, string password)
                 // if they just passed user then append the machine name here.
                 if (null != userName)
                 {
-                    int index = userName.IndexOf('\\');
-                    if (index == -1)
+                    if (!userName.Contains('\\'))
```

Comment:
This DLL is compiled against netstandard2.0 and netfx, which don't contain `string.Contains(char)`. This will lead to compilation failures.